docs/docs/outputs/logging.md

```diff
@@ -1,55 +1,90 @@
-Configure log severity:
+# Defining Tracee Logs
+
+This section showcases how to configure the way Tracee collects logs. This is done through the configuration file. For more information, have a look at the respective section in the [installation guide.](../install/index.md)
```

Comment:
> how to configure the way Tracee collects logs

no, this is about Tracee's diagnostic log level. we need to rewrite the intro here to be correct, and to explain the motivation (troubleshooting).